doc/starlight/src/content.config.ts

```diff
@@ -1,13 +1,11 @@
 import { defineCollection, z } from "astro:content";
-import { glob, type Loader } from "astro/loaders";
+import { type Loader } from "astro/loaders";
 import { docsSchema } from "@astrojs/starlight/schema";
 import { promises as fs } from "node:fs";
+import { docsLoader } from "@astrojs/starlight/loaders";
```

Comment:
```suggestion
import { docsLoader } from "@astrojs/starlight/loaders";
import { docsSchema } from "@astrojs/starlight/schema";
import { promises as fs } from "node:fs";
```
Makes my brain happier when they are grouped together :)